doc/guides/west/manifest.rst

```diff
@@ -1167,3 +1237,17 @@ or fails with an error:
    west manifest --validate
 
 The error message can help diagnose errors.
+
+.. _west-manifest-path:
+
+Get the manifest path
+=====================
+
+The ``--path`` action prints the path to the top level manifest file:
+
+.. code-block:: none
+
+   west manifest --path
+
+The output is something like ``/path/to/workspace/west.yml``. The path format
+depends on your operating system.
```

Comment:
So that means we now have:
`west topdir` that prints abs path posix style.
`west list` that prints relative path from workspace root in posix style
`west list --path={abspath}` printing abspath in host style (`\` on windows)
`west list --path={posixpath}` printing abspath in posix style

So should this command actually have printed relative posix path per default, and the support a `--format={abspath}` and `--format={posixpath}` ?

Reply:
> So should this command actually have printed relative posix path per default, and the support a `--format={abspath}` and `--format={posixpath}` ?

I'm confused as we already discussed offline that I think `west topdir` should always have printed a host path.

This hasn't been released yet so there's still time to change it, but please specify clearly whether or not you are asking for a change in west 0.8.

Reply:
> I'm confused as we already discussed offline that I think west topdir 

partly discussed, cause offline we only discussed `topdir`.
With this PR I actually noticed that `west list` also prints posix style per default.

Both `topdir` and `list` does posix per default, so there is also the consistency among commands to consider.

I agree on the principle that most correct is to use host style (and then allow a `--format={}` option like `west list` does).
(which means that both `topdir` and `list` should change default in future)

I made the comment because `list` was not path of earlier talks, but also to give others a chance to raise their opinion.
At the end we can refer back here to any decisions taken.

Reply:
> I'm not seeing a clear "I'm asking for a change" 

because i'm not requiring a change.

> Valid topic for discussion, for sure, but I'm hoping to focus on issues we need to resolve for documenting this release in this PR.

Accepted.